auth/realm/base/src/test/java/org/wildfly/security/auth/realm/AggregateRealmTest.java

```diff
@@ -16,18 +16,16 @@
 
 package org.wildfly.security.auth.realm;
 
-import static org.junit.Assert.assertEquals;
```

Comment:
I think these imports can be added back.

Reply:
Fixed